File: server_mybatis/fli/src/main/java/com/leepark/fli/model/service/PlaylistService.java
package com.leepark.fli.model.service;

import java.util.List;

import com.leepark.fli.model.dto.Playlist;
// Service -> DAO의 DB 접근 기본 메서드를 조립해서 새로운 기능을 구현하는 것
public interface PlaylistService {
	Integer insert(Playlist playlist);
	List<Playlist> selectAll(String userId, Integer userSns);
	Integer delete(Integer playlistId);
//	List<PlaylistInfo> selectAll(String userId, Integer userSns);
}

File: client/fli/app/src/main/java/com/ssafy/fli/network/PlayListService.kt
package com.ssafy.fli.network

import com.ssafy.fli.model.PlayList
import com.ssafy.fli.model.PlayListBody
import retrofit2.http.*

interface PlayListService {
    @GET("playlist/{userId}/{userSns}")
    suspend fun getAllPlayList(
        @Path("userId") userId: String,
        @Path("userSns") userSns: Int
    ): List<PlayList>

    @POST("playlist/insert")
    suspend fun insertPlayList(@Body playlist: PlayListBody): Int

    @DELETE("playlist/delete/{playlistId}")
    suspend fun deletePlayList(@Path("playlistId") playListId: Int): Int
}